app/src/main/res/layout/item_today_screen.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="#F0EFFF"
+    android:focusable="true">
+<LinearLayout
+    android:id="@+id/linearLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+   >
+
+    <LinearLayout
+        android:layout_width="342dp"
+        android:layout_height="42dp"
+        android:layout_marginStart="24dp"
+        android:layout_marginEnd="40dp"
+        android:layout_marginTop="10dp"
+        android:orientation="vertical">
+    <LinearLayout
+        android:id="@+id/home_screen_second_part"
+        android:layout_width="326dp"
+        android:layout_height="32dp"
+        android:orientation="horizontal"
+        android:layout_marginBottom="10dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <TextView
+            android:id="@+id/hourTextView"
+            android:layout_width="39dp"
+            android:layout_height="18dp"
+            android:layout_marginTop="7dp"
+            android:layout_marginBottom="7dp"
+            android:text="@string/some_id"
+            android:textAppearance="@style/some_id"
+
+            />
+
+        <ImageView
+            android:id="@+id/weatherImageView"
+            android:layout_width="24.61dp"
+            android:layout_height="24.42dp"
+            android:layout_marginTop="3.76dp"
+            android:layout_marginBottom="3.82dp"
```

Comment:
?? Displays render 2 pixels in 2 pixels so if you have .76 or .82 this dimension is rendered as 4dp. So avoid using those dimension with decimals.